src/instructlab/cli/rag/ingest.py

```diff
@@ -0,0 +1,91 @@
+# Standard
+import logging
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.defaults import DEFAULTS
+from instructlab.rag.taxonomy_utils import lookup_processed_documents_folder
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option(
+    "--document-store-uri",
+    "uri",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_class="rag",
+    config_sections="document_store",
+)
+@click.option(
+    "--document-store-collection-name",
+    "collection_name",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_class="rag",
+    config_sections="document_store",
+)
+@click.option(
+    "--embedding-model-path",
+    "embedding_model_path",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_class="rag",
+    config_sections="embedding_model",
+)
+@click.option(
+    "--input-dir",
+    required=False,
+    type=click.Path(
+        exists=True,
+        dir_okay=True,
+        file_okay=False,
+    ),
+    help="Directory where pre-processed documents are located.",
+)
+@click.pass_context
+@clickext.display_params
+def ingest(
+    ctx,
+    uri,
+    collection_name,
+    embedding_model_path,
+    input_dir,
+):
+    """The embedding ingestion pipeline"""
+    logger.debug(f"Document Store: {collection_name} @ {uri}")
+    logger.debug(f"Embedding model: {embedding_model_path}")
+
+    if input_dir is None:
+        output_dir = ctx.obj.config.generate.output_dir
+        if output_dir is None:
+            output_dir = DEFAULTS.DATASETS_DIR
+            logger.debug(
```

Comment:
Could we make this a `warning` instead? My thought is that users should be aware a different directory is being written to than they might expect.